templates/_includes/nav.html

```diff
@@ -15,9 +15,9 @@
         <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
       </hx-disclosure>
       <hx-reveal id="{{ navItem.name | slugify }}" {% if page.url contains currentNav %}open{% endif %}>   
-      {%- if currentNav == "style" -%}
+      {%- if currentNav == "style" or currentNav == "resources" -%}
         {%- for article in site.docs -%}
-          {%- if article.category == "Style" -%}
+          {%- if article.category | slugify == currentNav" -%}
```

Comment:
Trailing extra `"` at the end can be removed.

Reply:
👍